.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and verify PR Title
```

Comment:
```suggestion
name: Semantic PR title
```
Also lets rename the workflow to `semantic-pr-title` as we are only checking PR title in this action

Reply:
done